servicefabric/2016-01-28/swagger/servicefabric.json

```diff
@@ -2289,7 +2296,7 @@
             "name": "PartitionKeyType",
             "in": "query",
             "description": "The type of the partition key",
-            "type": "integer",
+            "type": "string",
```

Comment:
Drawing some attention to this change (since it isn't mentioned in the PR title).
Could this be modeled as an enum? What are valid types?